src/components/Activities/DateCard.js

```diff
@@ -0,0 +1,50 @@
+import { getActivitiesList } from '../../services/activitiesApi';
+import styled from 'styled-components';
+import { months, daysOfWeek } from '../../helpers/days';
+import { toast } from 'react-toastify';
+
+let numberId = -1;
+
+export default function DateCard({ activityDate, token, dateId, setSelected }) {
+  const dayOfWeek = activityDate.split(',')[0];
+  const dayOfMonth = activityDate.split(',')[1].split(' ')[1];
+  const month = activityDate.split(',')[1].split(' ')[2];
+
+  async function selectDate(date) {
+    setSelected(dateId);
+    numberId = dateId;
+
+    try {
+      const response = await getActivitiesList(token, date);
+      console.log(response);
```

Comment:
remove console.log

Reply:
Tinha deixado só pra quem for listar as atividades ver por onde chega